### PR TITLE
Add Recruiter Signup

### PR DESCRIPTION
**Pull request by DNS-404**

### CODE REVIEW

File: recruiter/templates/footer.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+  <!-- jQuery, Popper.js, Bootstrap JS -->
+  <script src="../scripts/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+  <script src="../scripts/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>
+  <script src="../scripts/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>
+  <script src="../scripts/studentProfile.js"></script>
```
Comment (harsh-jindal) [inline recruiter/templates/footer.php]: Not required.

Comment (harsh-jindal) [inline recruiter/templates/footer.php]: Not required.

File: recruiter/recruiterSignup.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      $verify_password = mysqli_real_escape_string($dbc, trim($_POST['confirm-pwd']));
+      if(!empty($company_id) && !empty($company_name) && $company_category!="0" &&
+        !empty($hr_name) && !empty($hr_email) && 
+        !empty($password) && !empty($verify_password) &&
+        ($verify_password == $password)){
+          // Check if company_id is available
+          $query = "SELECT * FROM recruiters WHERE company_id = '$company_id'";
+          $data = mysqli_query($dbc, $query);
+          if(mysqli_num_rows($data) == 0){
+            // company_id is available
```
Comment (harsh-jindal) [inline recruiter/recruiterSignup.php]: *not available

File: recruiter/recruiterLogin.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        </div>
+        <div class="form-group row">
+          <div class="col-sm-10">
+            <button type="submit" class="btn btn-primary" name="submit">Log in</button>
+          </div>
+        </div>
+        <div class="form-group row">
+            <div class="col-sm-10">
+            <div>
+                <label>New user? Signup <a href="./recruiterSignup.php">here</a></label>
```
Comment (harsh-jindal) [inline recruiter/recruiterLogin.php]: We could write something other than 'user'

File: recruiter/recruiterSignup.php (hunk 2 above)
Comment (DNS-404) [reply recruiter/recruiterSignup.php]: This is the column that tells that company id is not already used, so it's available.

**Pull request merged**